content/20.pharmaceuticals.md

```diff
@@ -257,43 +256,46 @@ Dexamethasone remains the major success story from this category of repurposed d
+mAbs can be designed for a particular virus, and significant advances have been made in the speed at which new mAbs can be identified and produced.
+At the time of the SARS and MERS epidemics, interest in mAbs to reduce infection was never realized [@doi:10.1093/infdis/jiw080; @doi:10.1016/j.antiviral.2017.03.025], but this allowed for mAbs to quickly be considered among the top candidates against COVID-19.
 
 ##### Biologics and the Innate Immune Response
 
 Deaths from COVID-19 often occur when inflammation becomes dysregulated following an immune response to the SARS-CoV-2 virus.
 Therefore, one potential approach to reducing COVID-19 mortality rates is to manage the inflammatory response in severely ill patients.
 One candidate therapeutic identified that uses this mechanism is tocilizumab (TCZ).
-TCZ is a monoclonal receptor antibody that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 [@doi:10.1101/cshperspect.a016295].
+TCZ is a mAb that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 [@doi:10.1101/cshperspect.a016295].
 IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, which is comprised by immune system regulators that are primarily responsible for immune cell differentiation.
 Often used to treat chronic inflammatory conditions such as rheumatoid arthritis [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease.
 It has also been approved to treat CRS caused by CAR-T treatments [@dailymed-tocilizumab].
 While the secretion of IL-6 can be associated with chronic conditions, IL-6 is a key player in the innate immune response and is secreted by macrophages in response to the detection of pathogen-associated molecular patterns and damage-associated molecular patterns [@doi:10.1101/cshperspect.a016295].
 An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19, and patients who ultimately died had higher IL-6 levels at admission than those who recovered [@doi:10/ggnxb3].
 Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately died [@doi:10/ggnxb3].
 
-Currently, TCZ is being administered either as an intervention or as concomitant medication in {{ebm_tocilizumab_ct}} interventional COVID-19 clinical trials (Figure @fig:ebm-trials).
+Currently, TCZ is being administered either as an intervention or as concomitant medication in {{ebm_tocilizumab_ct}} interventional COVID-19 clinical trials (Figure @fig:ebm-trials). <!---Not sure what this means. Does “concomitant medication” mean it’s part of a combination or that it’s not at all part of the intervention, but patients are allowed to continue taking it (eg it’s part of the standard of care)?-->
```

Comment:
Looking into this, it seems like they are actually just meds the participants happen to be taking! I don't think those would be tracked by EBM Data Lab although I don't see the term in their web resources. I can look into this more tomorrow but also tagging @byrdjb and @agitter in case you have any thoughts!

Reply:
Sorry, I'm not sure either